### PR TITLE
Test and add RANGE parameter

### PR DESCRIPTION
**Pull request by niccokunzmann**
This adds to Issue #75.

### CODE REVIEW

**Comment by niccokunzmann** [comment]
#171 does not solve the problem. The tests still fail.

File: test/calendars/issue_75_range_parameter.ics [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+DTSTART:20240901T120000Z
+DTEND:20240901T140000Z
+RRULE:FREQ=DAILY;INTERVAL=2;UNTIL=20250920
+RDATE:20240924T090000Z
+SEQUENCE:0
+SUMMARY:ORIGINAL EVENT
+END:VEVENT
+BEGIN:VEVENT
+UID:210
+RECURRENCE-ID;RANGE=THISANDFUTURE:20240913T090000Z
```
Comment (jabiertxof) [inline test/calendars/issue_75_range_parameter.ics:15]: this must be `RECURRENCE-ID;RANGE=THISANDFUTURE:20240913T120000Z`

File: test/calendars/issue_75_range_parameter.ics [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+DTSTART:20240913T090000Z
+DTEND:20240913T160000Z
+SEQUENCE:1
+SUMMARY:MODIFIED EVENT
+END:VEVENT
+BEGIN:VEVENT
+UID:210
+RECURRENCE-ID;RANGE=THISANDFUTURE:20240923T090000Z
```
Comment (jabiertxof) [inline test/calendars/issue_75_range_parameter.ics:23]: this must be RECURRENCE-ID;RANGE=THISANDFUTURE:20240923T120000Z

**Comment by niccokunzmann** [comment]
Yes, I see it. You included the tests! Nice!

**Comment by niccokunzmann** [comment]
#181 does this and more

**Pull request closed without merging**